src/source-netmap.c

```diff
@@ -542,6 +542,8 @@ static void NetmapReleasePacket(Packet *p)
 {
     NetmapThreadVars *ntv = (NetmapThreadVars *)p->netmap_v.ntv;
 
+    BUG_ON(p->pkt_src == PKT_SRC_WIRE);
```

Comment:
I don't understand this, why is it a bug to have the `PKT_SRC_WIRE` value here?

Reply:
This is my typo, of course it should be BUG_ON (p-> pkt_src! = PKT_SRC_WIRE); I added a commit to fix this.